FreeRTOS/Demo/RISC-V-ISP/lib.c

```diff
@@ -0,0 +1,2 @@
+// Tell vTaskApplicationHook to 
```

Comment:
Since we don't have to deal with main_blinky anymore, we shouldn't need this define